### ISSUE TITLE
Darkmoon card: Volcano Not proccing

### ISSUE THREAD

**Issue by MoireMoire** [NRG-Wow/NRGWow-Bugtracker#131]
**Reporter(Player/Character) Name**: Moire

**Current behaviour**: Not even one proc 

**Expected behaviour**: The trinket has 45 secs ICD and somewhat around 30% proc rate

**Steps to reproduce the problem**: 

1. equip the trinket
2. go to a dummy
3. do damage
http://www.wowhead.com/item=62047/darkmoon-card-volcano#comments since you don't trust

**Comment by Wilcalaf** [comment]
Tested on a mage, its working as intended. what clase are you using?

**Comment by MoireMoire** [comment]
Moonkin druid, 15M damage done so far, not even 1 proc.

**Comment by MoireMoire** [comment]
Please check on every class because clearly there is something off.

**Comment by Illusie** [comment]
It's should not proc if you are not standing in specific yard from the Target.

**Comment by MoireMoire** [comment]
What the fuck are you saying dude, i used darkmoon cards for years and they used to proc if i was standing inside the boss, mid or max range with no difference.

**Comment by Illusie** [comment]
http://www.wowhead.com/spell=89091/volcanic-destruction 
Range | 20 yards (Short)

to proc the trinket u need stand 20yard or closer.

**Comment by Wilcalaf** [comment]
Although other Pservers are not valid references
Atlantiss had the same bug report
https://github.com/Atlantiss/BugTracker/issues/2997

Seems like its indeed range.

**Comment by Wilcalaf** [comment]
They also talk about it on Hades
https://cata-twinhead.twinstar.cz/?issue=2796

If you use google translate you can find that they talk about the same issue saying that they cannot find references that its **not** supposed to be like that....

**Comment by MoireMoire** [comment]
Yeah, seems like that if i stand too far from the  boss i'm not proccing the trinket.

**Comment by MoireMoire** [comment]
Well, i played retail, i always raided as a caster and i can swear you it never happened. It's illogical that a caster has to stand melee range to make a caster trinket proc.

**Comment by Illusie** [comment]
The trinket is called 
Darkmoon Card: Volcano - http://cata-shoot.tauri.hu/?item=62047

what the trinket proc?

Volcanic Destruction - http://cata-shoot.tauri.hu/?spell=89091

Darkmoon Card: Volcano description:
When dealing damage with spells, you have a chance to deal 899 to 1500 additional Fire damage to the target **and gain 1600 Intellect for 12 sec.**

Volcanic Destruction description:
Deals 899 to 1500 Fire damage to an enemy and increases your **Intellect by 1600 for 12 sec**.

in other words to gain 1600 intellect you need proc the spell, and the spell is what gives you 1600 intellect.
That spell has 20 yard max. 
above 20 yard it won't proc = you wont gain 1600 intellect.

http://cata-shoot.tauri.hu/?spell=89091
Range | 20 yards (Twenty yards)
http://www.wowhead.com/spell=89091/volcanic-destruction
Range | 20 yards (Short)

I wish I helped.

**Comment by Wilcalaf** [comment]
@MoireMoire 
We have decided to remove the 20yards limitations due to lack of valid references about it (other private servers are usually not a valid refernce) and simple logic applied to the case.
I think its safe to assume that there was no limitation on retail and the best proof to it is the lack of reference to it (all reference come from private servers) which is pretty simple, if it was limited to 20 yards on retail then we would find ton of topics of people either complaining or asking why it doesnt work. but there is none to be find except for someone asking about it on PTR before it was released which is also just a question of speculation because on wowhead it had the 20 yard flag.

It will be applied on next patch

Thanks @Illusie for the help and the reference that help us as well reach to the conclusion

**Comment by Illusie** [comment]
I spoke with @Wilcalaf in private.
wowhead and tauri-wow in this case cannot be good source to relay on.
since I didn't played retail I don't have the knowledge to provide except from other pservers
as I do in vast of my reports with trying to provide links from retail or data that other provided.

I checked few cata servers and its divided about how the trinket works.
we know Moonwell Chalice is range dependent trinket but I think except that one there no other.

Since I'm not trying to retarding or trolling around and only help to improve NRG. I told Wilcalaf till there is not a unequivocal evidence.
~~(Except maybe watching Retail casters during those raids with the trinket.) which im lazy to do.~~

I been asked few people that played retail include Dev from another pserver.
the answers were divided aswell with advantage to no range based.

Because all of those things. And just because its not really a gamebreaking issue.
so After a deeeeeep search. I can say clearly I was wrong.
I prefer to go with the flow and I go back in everything I said here.

Have fun and chill ~

Edit:
I checked the comments in Twinstar issue
https://www.youtube.com/watch?v=7nHIBtHiMp0 1:05
seems like its more than 20yard

https://www.youtube.com/watch?v=DGMXqvGGSkk
0:22

https://www.mmo-champion.com/threads/828341-Darkmoon-Card-Volcano-PTR

P.S
Going to delete my messages tomorrow .

**Comment by Wilcalaf** [comment]
The range has been removed from the trinket, its on live server now.
